README.md

```diff
@@ -405,3 +406,33 @@ export NGROK_AUTHTOKEN="YOUR_AUTHTOKEN_HERE"; make test="-k TEST_CLASS.NAME_OF_T
 ```
 
 [See the MakeFile for more examples](./Makefile)
+
+### HTTP2 
+
+The examples include a minimal `hypercorn` HTTP/2 example if you run `make http2`. You can curl the endpoint logged with `INFO:ngrok.listener:Created` and verify the HTTP/2 response from `hypercorn`.
```

Comment:
Could you also add a link to hypercorn in the frameworks section:
https://github.com/ngrok/ngrok-python/blob/main/README.md#frameworks